2023-08-11 16:52:39,711 INFO    MainThread:69122 [internal.py:wandb_internal():90] W&B internal server running at pid: 69122, started at: 2023-08-11 16:52:39.711577
2023-08-11 16:52:39,713 DEBUG   HandlerThread:69122 [handler.py:handle_request():141] handle_request: run_start
2023-08-11 16:52:39,714 INFO    WriterThread:69122 [datastore.py:open_for_write():75] open: /home/nicole/marl-emecom/src/experiments_anastassacos/wandb/offline-run-20230811_165239-2rfae3su/run-2rfae3su.wandb
2023-08-11 16:52:39,715 DEBUG   HandlerThread:69122 [meta.py:__init__():35] meta init
2023-08-11 16:52:39,715 DEBUG   HandlerThread:69122 [meta.py:__init__():49] meta init done
2023-08-11 16:52:39,715 DEBUG   HandlerThread:69122 [meta.py:probe():209] probe
2023-08-11 16:52:39,716 DEBUG   HandlerThread:69122 [meta.py:_setup_git():199] setup git
2023-08-11 16:52:39,723 DEBUG   HandlerThread:69122 [meta.py:_setup_git():206] setup git done
2023-08-11 16:52:39,723 DEBUG   HandlerThread:69122 [meta.py:_save_pip():53] save pip
2023-08-11 16:52:39,724 DEBUG   HandlerThread:69122 [meta.py:_save_pip():67] save pip done
2023-08-11 16:52:39,724 DEBUG   HandlerThread:69122 [meta.py:probe():247] probe done
2023-08-11 16:52:44,685 WARNING MainThread:69122 [internal.py:wandb_internal():153] Internal process interrupt: 1
2023-08-11 16:52:44,888 WARNING MainThread:69122 [internal.py:wandb_internal():153] Internal process interrupt: 2
2023-08-11 16:52:44,888 ERROR   MainThread:69122 [internal.py:wandb_internal():156] Internal process interrupted.
2023-08-11 16:52:45,695 INFO    WriterThread:69122 [datastore.py:close():279] close: /home/nicole/marl-emecom/src/experiments_anastassacos/wandb/offline-run-20230811_165239-2rfae3su/run-2rfae3su.wandb
2023-08-11 16:52:45,696 INFO    HandlerThread:69122 [handler.py:finish():806] shutting down handler
2023-08-11 16:52:45,715 INFO    SenderThread:69122 [sender.py:finish():1111] shutting down sender
2023-08-11 16:52:45,716 INFO    MainThread:69122 [internal.py:handle_exit():80] Internal process exited
